webapp/locales/en.json

```diff
@@ -482,11 +482,67 @@
     "reports": {
       "empty": "Congratulations, nothing to moderate.",
       "name": "Reports",
+      "status": "Current status",
+      "content": "Content",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-30T10:14:28Z" title="Saturday, November 30th 2019, 11:14:28 am +01:00">Nov 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -482,7 +482,7 @@
     "reports": {
       "empty": "Congratulations, nothing to moderate.",
       "name": "Reports",
-      "typeRowHeadline": "Type",
+      "typeRowHeadline": "Current status",
```


I'd have to check, but I guess this is not the `Current status`... by `Type`, I think @Tirokk was referring to the `type` that used to be returned from the `backend` as in the `type of Resource that was reported`... in the `NotificationsTable`, I didn't have a header for this column...
I'll see what @alina-beck thinks about the `layout` though, since I trust her judgement more than any of ours :stuck_out_tongue_closed_eyes: